pkg/machine/qemu/options_darwin.go

```diff
@@ -0,0 +1,15 @@
+package qemu
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+)
+
+func getDataDir() (string, error) {
+	tmpDir, ok := os.LookupEnv("TMPDIR")
```

Comment:
Wait - we're putting our data in /tmp? This doesn't seem right. Shouldn't we be using /home?

Reply:
first, look at the file name ... it is darwin which does not have the directories we would like.  The method name is misleading as it says data dir but it is really the socket location.

Reply:
Then it should be renamed so people adding to this later don't see `getDataDir()` and use it expecting persistence